trunk/makefile

```diff
@@ -7,6 +7,7 @@
 COMPLEX    ?= 0
 DEBUG      ?= 0
 
+HP3D_USE_OPENMP    ?= 0
```

Comment:
This change will require the default `m_options` files to be changed so that OpenMP can be switched on/off within the user-defined `m_options`.